### PR TITLE
Add exportable IdentifiesFailedError, vendor in some other SC codes

### PR DESCRIPTION
**Pull request by mlsteele**
Add `IdentifiesFailedError` for when one more identifies fail when identifying an iteam. KBFS will see this error and know that of the `ResolveIdentifyImplicitTeamRes` is valid and contains a trackbreaks map.

code reserved here: https://github.com/keybase/keybase/pull/1652

### CODE REVIEW

File: go/libkb/errors.go [hunk 1]
```diff
@@ -1358,6 +1358,15 @@ func (e IdentifyFailedError) Error() string {
 
 //=============================================================================
 
+type IdentifiesFailedError struct {
+}
+
+func (e IdentifiesFailedError) Error() string {
+	return fmt.Sprintf("one or more identifies failed")
```
Comment (patrickxb) [inline go/libkb/errors.go]: just `return "one or more..."`

Comment (mlsteele) [reply go/libkb/errors.go]: ok, why?

Comment (patrickxb) [reply go/libkb/errors.go]: it's just a string, so just return it...there isn't any formatting of it going on.

Comment (mlsteele) [reply go/libkb/errors.go]: oh, heh, I thought you meant make it an ellipsis

**Pull request merged**